test/test_nn.py

```diff
@@ -8954,7 +8686,9 @@ def __call__(self, input):
 add_test(NewModuleTest(
     constructor=lambda: _AdaptiveLogSoftmaxWithLoss(16, 10, [2, 6]),
     input_size=(4, 16),
-    fullname='AdaptiveLogSoftmax'))
+    fullname='AdaptiveLogSoftmax',
```

Comment:
Is this function also affected by the flag / use TF32? It wasn't mentioned.

Reply:
Yes, and from the test yes. Docs is updated to mention this.